tests/run/skip_limited_api_helper.pxi

```diff
@@ -5,7 +5,7 @@ def skip_if_limited_api(why, *, min_runtime_version=None):
     import sys
     def dec(f):
         if (CYTHON_COMPILING_IN_LIMITED_API and
-                (min_runtime_version is None or sys.version_info >= min_runtime_version)):
+                (min_runtime_version is None or sys.version_info <= min_runtime_version)):
```

Comment:
`<` only, maybe? I guess is doesn't matter in practice sine the full version tuple will always be greater than the corresponding two-part `min_runtime_version`, but I think it'd still be more correct.